plugins/TestRunner/tests/System/CheckDirectDependencyUseCommandTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\TestRunner\tests\System;
+
+use Piwik\Plugins\TestRunner\Commands\CheckDirectDependencyUse;
+use Piwik\Tests\Framework\TestCase\SystemTestCase;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\NullOutput;
+
+/**
+ * @group TestRunner
+ * @group TestRunner
+ * @group Plugins
+ */
+class CheckDirectDependencyUseCommandTest extends SystemTestCase
```

Comment:
I reckon we also need a check for a mock plugin which will fail the test, i.e. return some uses of direct dependencies. Without that we can't be sure it will return anything or not.

Reply:
@michalkleiner Will add a test for same

Reply:
@michalkleiner Added please check